etc/httpd/global.conf

```diff
@@ -882,8 +881,9 @@ RewriteRule ^/(contributor-data/[a-z]+.json)$ /b/$1 [L,PT]
 # bug 1108828
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?firefox/feedback(/?.*)$ /b/$1firefox/feedback$2 [PT]
 
+# bug 1148127
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?products(/?)$ /$1firefox/products/ [L,R=302]
```

Comment:
FTR, this reordering wasn't actually necessary, just the [removal of line 284](https://github.com/mozilla/bedrock/pull/2885/files#diff-cc7713eb3a36ef9c90c2f1b99f8966e8L284) -- it doesn't hurt anything to reorder this though, so don't worry about undoing it :)